README.md

```diff
@@ -46,6 +46,21 @@ $ cd ~/workspace/my_service/service_fabrik_backup_restore
 $ pip3 install -r requirements.txt
 ```
 
+### DEVELOPING WITH THE LIBRARY
+We are using [pytest](https://pytest.readthedocs.io/en/2.7.3/index.html) as a unit tests framework. You can use setup tool to run tests with multiple options:
```

Comment:
Change the doc version to 3.5.1 as we are going to use it

https://pytest.readthedocs.io/en/3.5.1/index.html

Reply:
Done.